helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/CompatibleSpecificRecordBuilderBase.java

```diff
@@ -86,7 +86,7 @@ protected static boolean isValidValue(Schema.Field f, Object value) {
     return false;
   }
 
-
+  @SuppressWarnings("unchecked")
   protected  <T> T deepCopyField (Object inputValue, Schema fieldSchema,
```

Comment:
Please add the test case which produces this error as well.

Reply:
added test